### DISCUSSION TITLE
GRUPO TEAMS

### DISCUSSION THREAD

**Discussion by LuisCastillo08** [Launch-X-Latam/OnBoarding#8] [Q&A]
¿Ya los aceptaron en el grupo de Teams?
Mande la solicitud y no me han aceptado:(

**Comment by ghost** [comment]
a mi tampoco me acetan hasta ahora 

![image](https://user-images.githubusercontent.com/114446633/192400930-cb626ea0-965c-4f5e-881a-7f8694c8527c.png)

**Reply by Gl096** [reply]
me sucede lo mismo ;(

**Reply by JassonLr** [reply]
x2

**Reply by jeantifi1** [reply]
También me pasa lo mismo.

**Comment by Capirossi** [comment]
Hola, y donde está ese acceso?, estoy demasiado perdido en este formato de estudio.

**Reply by devonent** [reply]
Lo puedesn encontrar en la sección de "[Accede al grupo de la Misión FrontEnd en Microsoft Teams](https://github.com/Launch-X-Latam/OnBoarding#accede-al-grupo-de-la-misi%C3%B3n-frontend-en-microsoft-teams)", está ubicado en este repositorio.

El link directo a la solicitud es [este](https://forms.office.com/r/Ddr5fxJbNk).
